packages/foundations-ts-definitions/types/organisations-schema.ts

```diff
@@ -0,0 +1,717 @@
+/**
```

Comment:
You should use this file 👍 

Can you copy over to the packages/reapit-connect-management/src/types folder and reference from there please?